README.md

```diff
@@ -1,192 +1,144 @@
-  - [Getting Started](#getting-started)
-    - [Prerequisites](#prerequisites)
-    - [Cluster Creation](#cluster-creation)
-    - [Managed OpenStack Security Groups](#managed-openstack-security-groups)
-    - [Interacting with your cluster](#interacting-with-your-cluster)
-    - [Cluster Deletion](#cluster-deletion)
-    - [Trouble shooting](#trouble-shooting)
-  - [Contributing](#contributing)
+[![Go Report Card](https://goreportcard.com/badge/kubernetes-sigs/cluster-api-provider-openstack)](https://goreportcard.com/report/kubernetes-sigs/cluster-api-provider-openstack)
 
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+------
 
-# Kubernetes cluster-api-provider-openstack Project
+Kubernetes-native declarative infrastructure for OpenStack.
 
-This repository hosts a concrete implementation of an OpenStack provider for the [cluster-api project](https://github.com/kubernetes-sigs/cluster-api).
+## What is the Cluster API Provider OpenStack
 
-## Community, discussion, contribution, and support
+The [Cluster API][cluster_api] brings
+declarative, Kubernetes-style APIs to cluster creation, configuration and
+management.
 
-Learn how to engage with the Kubernetes community on the [community page](http://kubernetes.io/community/).
+The API itself is shared across multiple cloud providers allowing for true OpenStack
+hybrid deployments of Kubernetes. It is built atop the lessons learned from
+previous cluster managers such as [kops][kops] and
+[kubicorn][kubicorn].
 
-You can reach the maintainers of this project at:
 
-- [#cluster-api-openstack on Kubernetes Slack](https://kubernetes.slack.com/messages/cluster-api-openstack)
-- [SIG-Cluster-Lifecycle Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle)
+## Launching a Kubernetes cluster on OpenStack
 
-### Code of conduct
+Check out the [getting started guide](./docs/getting-started.md) for launching a cluster on OpenStack. 
 
-Participation in the Kubernetes community is governed by the [Kubernetes Code of Conduct](code-of-conduct.md).
+## Features
+
+- Native Kubernetes manifests and API
+- Choice of Linux distribution (as long as a current cloud-init is available)
+- Support for single and multi-node control plane clusters
+- Deploy clusters with and without LBaaS available
+- Support for security groups
+- Doesn't use SSH for bootstrapping nodes
 
 ------
 
-## Compatibility with Cluster API, Kubernetes and OpenStack Versions
+## Compatibility with Cluster API and Kubernetes Versions
 
 This provider's versions are compatible with the following versions of Cluster API:
 
-||Cluster API v1alpha1 (v0.1)|
-|-|-|
-|OpenStack Provider v1alpha1 (ea309e7f)|✓|
+||Cluster API v1alpha1 (v0.1)|Cluster API v1alpha2 (v0.2)|
+|-|-|-|
+|OpenStack Provider v1alpha1 (release-0.1 branch)|✓||
+|OpenStack Provider v1alpha2 (v0.2)||✓|
```

Comment:
should we say master instead of v0.2?  or v0.2 branch?

Reply:
Because I wanted to release the 0.2 based on v1alpha2 and we shouldn't branch away a release-0.2 before v1alpha3 I would like to keep 0.2 (see also: https://github.com/kubernetes-sigs/cluster-api-provider-aws#compatibility-with-cluster-api-and-kubernetes-versions)

Reply:
ok, then why we need v0.2 ? as master actually works fine and we can have release-0.2 when v1alpha3 out..

Reply:
0.2 stands for the tags 0.2.0, 0.2.1, etc.. I think it's more precise to say CAPO 0.2 then something on the master branch. And it's the same as CAPA is doing

Reply:
0.2 stands for the tags make sense to me.